sdk/evaluation/azure-ai-evaluation/tests/unittests/test_evaluate.py

```diff
@@ -636,25 +635,23 @@ def test_optional_inputs_with_data(self, questions_file, questions_answers_basic
         only_question_results = evaluate(
             data=questions_file,
             evaluators={"half": HalfOptionalEval(), "opt": OptionalEval(), "no": NoInputEval()},
-            _use_pf_client=use_pf_client,
+            _use_pf_client=False,
         )  # type: ignore
 
         first_row_2 = only_question_results["rows"][0]
         assert first_row_2["outputs.half.half_score"] == 0
         assert first_row_2["outputs.opt.opt_score"] == 1
-        if use_pf_client:
-            assert first_row["outputs.no.no_score"] == 0
 
-    @pytest.mark.parametrize("use_pf_client", [True, False])
-    def test_optional_inputs_with_target(self, questions_file, questions_answers_basic_file, use_pf_client):
+    @pytest.mark.skip(reason="Breaking CI by crashing pytest somehow")
+    def test_optional_inputs_with_target(self, questions_file, questions_answers_basic_file):
         from test_evaluators.test_inputs_evaluators import EchoEval
 
         # Check that target overrides default inputs
         target_answer_results = evaluate(
             data=questions_file,
             target=_new_answer_target,
             evaluators={"echo": EchoEval()},
-            _use_pf_client=use_pf_client,
+            _use_pf_client=False,
```

Comment:
We don't need to address this in this PR, but if everyone agrees, I propose removing _use_pf_client. I don't see the necessity of having the code client anymore. Tagging @singankit

Reply:
I'll bring that up during the next standup